charts/astronomer/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: astronomer
-version: 0.28.0
+version: 0.28.1
```

Comment:
We are already at 0.28.2 now. This should be 0.28.3

Reply:
```suggestion
version: 0.28.3
```